### PR TITLE
Enable package signing on NuGet.Server .nupkgs

### PR DESCRIPTION
**Pull request by joelverhagen**
Address https://github.com/NuGet/Engineering/issues/1816

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -30,7 +30,7 @@ if (-not (Test-Path "$PSScriptRoot/build")) {
 # Enable TLS 1.2 since GitHub requires it.
 [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Tls12
 
-wget -UseBasicParsing -Uri "https://raw.githubusercontent.com/NuGet/ServerCommon/$BuildBranch/build/init.ps1" -OutFile "$PSScriptRoot/build/init.ps1"
+Invoke-WebRequest -UseBasicParsing -Uri "https://raw.githubusercontent.com/NuGet/ServerCommon/$BuildBranch/build/init.ps1" -OutFile "$PSScriptRoot/build/init.ps1"
```
Comment (scottbommarito) [inline build.ps1]: Aren't these equivalent? Any reason to use one over the other?

File: build/sign.proj [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (scottbommarito) [inline build/sign.proj]: Is this file something we'd want to share anywhere? I don't see anything `NuGet.Server` specific in here.

File: build.ps1 (hunk 1 above)
Comment (joelverhagen) [reply build.ps1]: Yes, equivalent. I was getting a warning from VS code.

File: build/sign.proj (hunk 2 above)
Comment (joelverhagen) [reply build/sign.proj]: it does vary per repo. This is essentially a list of tools needed during build

**Pull request merged**